core/lib/basic_types/src/protocol_version.rs

```diff
@@ -255,3 +278,106 @@ impl From<ProtocolVersionId> for VmVersion {
         }
     }
 }
+
+basic_type!(
+    /// Patch part of semantic protocol version.
+    VkPatch,
+    u16
```

Comment:
why not u32? The packing of the type allows to have u32 patch type, so not sure why not use it all

Reply:
I wanted to be consistent with ProtocolVersionId. Agree, better to use u32